tests/test_classifier.py

```diff
@@ -14,19 +14,19 @@ def test_classify(self, fs: FakeFilesystem) -> None:
         fs.add_real_directory('./testdata/test_model/test_instance_unreleased')
         c = Classifier('./testdata', 'test_model')
 
-        result = c.classify(['Where is my medical book?'])
+        result = c.classify(['Increase access to health care'])
 
         assert c.vocab is not None
         assert c.embedder is not None
         assert c.predictor is not None
 
         assert c.instance == 'test_instance'
-        print(result)
         assert result
         # result ~ [{topic: probability, topic2: probability, ...}, ...]
         for topic, _ in result[0].items():
             assert topic in c.vocab
-        assert result[0]['Right to education'] >= 0.7
+        assert len(result) == 1
+        assert result[0]['Right to health'] >= 0.5
```

Comment:
why such a low threshold? this should be a high-confidence sequence, right?

Reply:
the test still passes with an increased threshold. I just adapted that.